packages/platforms/browser/lib/routing-provider.ts

```diff
@@ -1,7 +1,7 @@
 import { isObject, type Span, type SpanOptions } from '@bugsnag/core-performance'
 
 export type RouteChangeSpanOptions = Omit<SpanOptions, 'isFirstClass'>
-export type StartRouteChangeCallback = (route: string, trigger: string, options?: RouteChangeSpanOptions) => Span
+export type StartRouteChangeCallback = (url: string, route: string, trigger: string, options?: RouteChangeSpanOptions) => Span
```

Comment:
If this accepts a `url` then it doesn't need `route` since we can call `resolveRoute` to get that

It also might be worth making it `URL | string`?